### PR TITLE
tested and working. un-needed values removed from templates

### PR DESCRIPTION
**Pull request by iamemilio**
@danmcp I could only test the non swift template. I ran hello and cal commands against it in my local oc cluster up and they returned fine.

i also added .vscode files to the git ignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,7 @@
 
 # Python egg metadata, regenerated from source files by setuptools.
 /*.egg-info
+
+#vs code stuff
+.vscode/
```
Comment (danmcp) [inline .gitignore]: Do this in your global git config instead.

Comment (iamemilio) [reply .gitignore]: should i roll it back

Comment (danmcp) [reply .gitignore]: yes

**Comment by iamemilio** [comment]
@danmcp the changes to the .gitignore were taken out

**Comment by danmcp** [comment]
@iamemilio Please squash the 2 commits.

**Pull request merged**